### PR TITLE
Refactor block

### PR DESCRIPTION
**Pull request by nhunzaker**
I added a `respondsTo` function to abstract a way some checks and make sure that `null` and `undefined` are handled. I also renamed "Move Down" and "Move Up" to "Move After" and "Move Before". This is because we will have components that display left to right and we need more generalized labels.

screen shot 2015-05-08 at 12 08 17 pm

### CODE REVIEW

File: src/utils/__tests__/respondsTo.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let respondsTo = require('../respondsTo')
+
+describe('Utils - respondsTo', function() {
+
+  it ('returns true when a key is a function', function() {
+    respondsTo({ test(){} }, 'test').should.equal(true)
+  })
+
+  it ('returns true when a key is not a function', function() {
```
Comment (solomonhawk) [inline src/utils/__tests__/respondsTo.test.js]: "False"

**Comment by solomonhawk** [comment]
Nice. :+1:

File: src/utils/__tests__/respondsTo.test.js (hunk 1 above)
Comment (nhunzaker) [inline src/utils/__tests__/respondsTo.test.js]: Oops, yes

Comment (nhunzaker) [inline src/utils/__tests__/respondsTo.test.js]: Fixed

**Comment by averyvery** [comment]
Cleaner and nice :+1:

**Pull request merged**